build/Platform.targets

```diff
@@ -0,0 +1,156 @@
+<Project>
+  <Import Project="common.props" />
```

Comment:
This feels odd and we should restructure the targets so they don't have to explicitly import props. Here, I needed to use a few version properties that were in dependency.props.